deon/deon.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
Do we need to add something like `from deon import create` to `__init__.py` so that we can import `deon.create` instead of `deon.deon.create`?